src/components/display/DesktopDisplay.tsx

```
@@ -1,84 +1,86 @@
-//                   </svg>
-//                   <span className="ml-2">GitHub</span>
-//                 </Link>
-//               </Button>
-//             )}
-//           </div>
-//         </div>
-//       </div>
-//     );
-//   }
+export default function DesktopDisplay({ currentVersion} : DesktopDisplayProp) {
+    {/* Desktop View: Maintain existing layout */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)